contracts/FlowFees.cdc

```diff
@@ -70,6 +71,59 @@ pub contract FlowFees {
         }
     }
 
+    // VerifyPayerBalanceResult is returned by the verifyPayersBalanceForTransactionExecution function
+    pub struct VerifyPayerBalanceResult {
+        // True if the payer has sufficient balance for the transaction execution to continue
+        pub let canExecuteTransaction: Bool
+        // The minimum payer balance required for the transaction execution to continue
+        pub let requiredBalance: UFix64
```

Comment:
So this balance does not take into account any potential execution effort? might be good to clarify that in the comment

Reply:
It does. It assumes that the execution effort is the maximum possible (the computation/gas limit set by the transaction).